extensions/wikia/UserLogin/js/UserSignupAjaxValidation.js

```diff
@@ -14,26 +14,20 @@
 	var UserSignupAjaxValidation = function (options) {
 		this.wikiaForm = options.wikiaForm;
 		this.inputsToValidate = options.inputsToValidate || [];
-		this.notEmptyFields = options.notEmptyFields || [];
 		this.submitButton = $(options.submitButton);
-
-		this.activateSubmit();
-	};
-
-	UserSignupAjaxValidation.prototype.activateSubmit = function () {
```

Comment:
We're no longer disabling the submit button until a form validates on the front end. We're just going to let users submit whenever they want, since back end validation will pick it up.